huaweicloud/services/acceptance/mpc/resource_huaweicloud_mpc_transcoding_template_group_test.go

```diff
@@ -0,0 +1,178 @@
+					resource.TestCheckResourceAttr(resourceName, "video_common.0.codec", "2"),
+					resource.TestCheckResourceAttr(resourceName, "video_common.0.profile", "4"),
+					resource.TestCheckResourceAttr(resourceName, "video_common.0.output_policy", "transcode"),
+					resource.TestCheckResourceAttr(resourceName, "videos.0.width", "1920"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testMPCTranscodingTemplateGroup_update(rNameUpdate),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "name", rNameUpdate),
+					resource.TestCheckResourceAttr(resourceName, "common.0.pvc", "false"),
+					resource.TestCheckResourceAttr(resourceName, "common.0.pack_type", "2"),
+					resource.TestCheckResourceAttr(resourceName, "audio.0.codec", "1"),
+					resource.TestCheckResourceAttr(resourceName, "video_common.0.codec", "1"),
+					resource.TestCheckResourceAttr(resourceName, "video_common.0.profile", "3"),
+					resource.TestCheckResourceAttr(resourceName, "videos.0.width", "3840"),
+					resource.TestCheckResourceAttr(resourceName, "videos.1.width", "2560"),
+				),
+			},
+		},
+	})
+}
+
+func testMPCTranscodingTemplateGroup_basic(rName string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_mpc_transcoding_template_group" "test" {
+  name = "%s"
+
+  common {
+    pvc           = true
+    dash_interval = 5
+    hls_interval  = 5
+    pack_type     = 1
+  }
+
+  audio {
+    bitrate       = 0
+    channels      = 2
+    codec         = 2
+    output_policy = "transcode"
+    sample_rate   = 1
+  }
+
+  video_common {
+    bframes_count        = 7
+    black_cut            = 0
+    codec                = 2
+    frame_rate           = 0
+    level                = 15
+    max_iframes_interval = 5
+    output_policy        = "transcode"
+    preset               = 1
+    profile              = 4
+    ref_frames_count     = 4
+  }
+
+  videos {
+    width   = 1920
+    height  = 1080
+    bitrate = 0
+  }
+}
+`, rName)
+}
+
+func testMPCTranscodingTemplateGroup_update(rName string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_mpc_transcoding_template_group" "test" {
+  name = "%s"
+
+  common {
+    pvc           = false
+    dash_interval = 5
+    hls_interval  = 5
+    pack_type     = 2
+  }
+
+  audio {
+    bitrate       = 0
+    channels      = 2
+    codec         = 1
+    output_policy = "transcode"
+    sample_rate   = 1
+  }
+
+  video_common {
+    bframes_count        = 7
+    black_cut            = 0
+    codec                = 1
+    frame_rate           = 0
+    level                = 15
+    max_iframes_interval = 5
+    output_policy        = "transcode"
+    preset               = 1
+    profile              = 3
+    ref_frames_count     = 4
+  }
+
+  videos {
+    width   = 3840
+    height  = 2160
+    bitrate = 0
```

Comment:
The default value also suggests checking, which can be checked by not writing it in the script and using `resource.TestCheckResourceAttr(resourceName, "videos.0.bitrate", "0"),`.

Reply:
no need to change